src/sugar/parse/ledger.ts

```diff
@@ -6,7 +6,7 @@ import { removeUndefined, rippleTimeToISOTime } from '../../utils'
 import parseTransaction from './transaction'
 
 export interface FormattedLedger {
-  // TODO: properties in type don't match response object. Fix!
```

Comment:
Why are we worrying about TODOs in files that are about to be deleted?

Reply:
Wasn't, just removed the `TODO` part and made a note that this will be removed.